rllib/policy/tests/test_sample_batch.py

```diff
@@ -95,6 +95,25 @@ def test_concat(self):
         concatd_2 = s1.concat(s2)
         check(concatd, concatd_2)
 
+    def test_concat_max_seq_len(self):
+        """Tests, SampleBatches.concat_samples() max_seq_len."""
+        s1 = SampleBatch({
+            "a": np.array([1, 2, 3]),
+            "b": {
+                "c": np.array([4, 5, 6])
+            },
+            SampleBatch.SEQ_LENS: [1, 2]
+        })
+        s2 = SampleBatch({
+            "a": np.array([2, 3, 4]),
+            "b": {
+                "c": np.array([5, 6, 7])
+            },
+            SampleBatch.SEQ_LENS: [3]
+        })
+        concatd = SampleBatch.concat_samples([s1, s2])
+        check(concatd.max_seq_len, 3)
```

Comment:
@gjoliver Instead of `check(concatd.max_seq_len, 3)` this could also be `check(concatd.max_seq_len, s2.max_seq_len)` if you like that better.